animal-sniffer/pom.xml

```diff
@@ -43,6 +43,11 @@
       <groupId>org.ow2.asm</groupId>
       <artifactId>asm</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.codehaus.mojo</groupId>
+      <artifactId>animal-sniffer-annotations</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
Is there a reason why `animal-sniffer-annotations` was previously not a dependency? It allows directly referring to the annotation type `IgnoreJRERequirement` in `SignatureChecker`.

Though if you want I can also revert this again.